sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosClientBuilder.java

```diff
@@ -59,8 +62,33 @@ public void enableSessionCapturing(boolean sessionCapturingOverride) {
      *
      * @return the session capturing override
      */
-    public boolean isSessionCapturingOverride() {
-        return this.sessionCapturingOverride;
+    public boolean isSessionCapturingOverrideEnabled() {
+        return this.sessionCapturingOverrideEnabled;
+    }
+
+    /**
+     * Enables transport client sharing allows. The default is false.
+     *
+     * When you have multiple instances of Cosmos Client in the same JVM interacting to multiple Cosmos accounts,
+     * enabling this allows connection sharing in Direct mode if possible between instances of Cosmos Client.
+     * @param transportClientSharingEnabled transport client sharing
+     * @return current cosmosClientBuilder
+     */
+    public CosmosClientBuilder setTransportClientSharingEnabled(boolean transportClientSharingEnabled) {
```

Comment:
>setTransportClientSharingEnabled [](start = 31, length = 32)

TransportClient is an implementation detail. How about name reflecting of connection sharing across clients?

Reply:
agree. I am changing to `setConnectionSharingAcrossClientsEnabled()`

Reply:
addressed.